### PR TITLE
Markdown version of Dockerhub readme updated for 0.4.0 container release

### PR DESCRIPTION
**Pull request by harrism**

### CODE REVIEW

File: dockerhub/README.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+1.  First launch an interactive session:
+
+        docker run --runtime=nvidia \
+                --rm -it \
+                -p 8888:8888 \
+                -p 8787:8787 \
+                -p 8786:8786 \
+                rapidsai/rapidsai:ubuntu1604_cuda92_py35
+
```
Comment (mt-jones) [inline dockerhub/README.md]: No code block?

File: dockerhub/README.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+## Common Errors
+
+*   Running out of device memory
+
+    *   ETL processes may involve creating many copies of data in device memory, resulting in sporadic spikes of memory utilization
+
+        *   Memory utilization which exceeds available device resources will cause the worker to crash
+        *   Because we've disabled experimental features that would enable the worker to recover, restart its work, or share its work with other workers, the error is silently propagated forward in the Dask's delayed task graph. This can manifest itself in the form of unusually short ETL times (sub-millisecond timescale)
+        *   An error may be raised by another routine referring to `NoneType` in `data` or similar
+        *   Note: it is possible to enable these _experimental_ features by changing the Dask `export` parameters in the `/rapids/utils/dask-setup.sh`:
```
Comment (mt-jones) [inline dockerhub/README.md]: Let's get rid of this experimental session.

I think there shouldn't be process `nanny`ing, yet. We don't support it.

File: dockerhub/README.md (hunk 1 above)
Comment (harrism) [reply dockerhub/README.md]: I just converted what you wrote on dockerhub from HTML to MD using a convertor.

File: dockerhub/README.md (hunk 2 above)
Comment (harrism) [reply dockerhub/README.md]: Removed.

File: dockerhub/README.md (hunk 1 above)
Comment (harrism) [reply dockerhub/README.md]: Hold on, this *is* a code block. Note the indentation? It should render as a code block.

Comment (mt-jones) [reply dockerhub/README.md]: My mistake. Thanks.

**Pull request merged**